### ISSUE TITLE
Join "Project Connected Home over IP"?

### ISSUE THREAD

**Issue by egorio** [mozilla-iot/wot#146]
It seems giants are trying to work on [something like WOT](https://www.connectedhomeip.com/).
It will be great if you join this Working Group.

https://zigbeealliance.org/news_and_articles/connectedhomeIP/

https://www.apple.com/newsroom/2019/12/amazon-apple-google-and-the-zigbee-alliance-to-develop-connectivity-standard/

https://developers.googleblog.com/2019/12/project-connected-home-over-ip.html

**Comment by kgiori** [comment]
Yes, we are keen on participating in efforts that improve interoperability. At the same time, we need to make sure that privacy and security (decentralized, not dependent on a few tech titans) is maintained in parallel with this new effort at interoperability. Optimistically looks positive to the existing WebThings Gateway since more standardization among smart home devices will gradually reduce the need for an add-on per brand... The Gateway still serves a useful function to allow legacy and new standards-based devices to co-exist in an interoperable way at the web layer.
